pkg/util/test/frequencyplans.go

```diff
@@ -417,3 +417,14 @@ var (
 func FrequencyPlan(id string) *frequencyplans.FrequencyPlan {
 	return Must(FrequencyPlanStore.GetByID(id)).(*frequencyplans.FrequencyPlan)
 }
+
+func StaticFrequencyPlans() map[string][]byte {
```

Comment:
Why not a variable that you can then reference in `FrequencyPlansFetcher` above ?